ana_scans.py

```diff
@@ -0,0 +1,227 @@
+#!/bin/env python
+
+def launchAna(args):
+  return launchAnaArgs(*args)
+
+def launchAnaArgs(anaType, cName, cType, scandate1, scandate2=None, ztrim=4.0, chConfigKnown=False, channels=False, panasonic=False):
+  import os
+  import subprocess
+  from subprocess import CalledProcessError
+  from anaInfo import ana_config
+  from gempython.utils.wrappers import runCommand
+
+  dataPath  = os.getenv('DATA_PATH')
+  dirPath   = ""
+  elogPath  = "%s/%s"%(os.getenv('ELOG_PATH'),scandate1)
+    
+  print "Analysis Requested: %s"%(anaType)
+
+  #Build Commands
+  cmd = ["python"]
```

Comment:
> Be added to the $PATH via cmsgemos/setup/paths.sh

*Definitely* not.  I have no interest in making `vfatqc-python-scripts` a dependency of `cmsgemos`
The currently it's probably done in a `.bashrc` or similar file (at P5 there's an env script for the `gempro` account)
> or to include some new script that is included in vfatqc-python-scripts that the user should execute like paths.sh is executed

Yeah, the best way is pretty simple: add a setup script to the repo to take care of setting up the environment for the package itself.